addon/components/polaris-choice-list.js

```diff
@@ -21,6 +22,12 @@ const CheckedChoice = ObjectProxy.extend({
  */
 export default Component.extend({
   tagName: 'fieldset',
+
+  attributeBindings: [
+    'hasError:aria-invalid',
+    'finalNameError:aria-described-by',
+  ],
```

Comment:
We seem to be missing the `id` attribute that's been [added](https://github.com/Shopify/polaris/compare/v2.2.0...v2.11.0?diff=split#diff-f9b830900da70d07e7b83b882b9e48d7R119) to the React version - the `fieldID` value we're passing to the new `polaris-inline-error` won't work without it